weed/command/filer_remote_gateway_buckets.go

```diff
@@ -30,10 +30,20 @@ func (option *RemoteGatewayOptions) followBucketUpdatesAndUploadToRemote(filerSo
 		return err
 	}
 
-	processEventFnWithOffset := pb.AddOffsetFunc(eachEntryFunc, 3*time.Second, func(counter int64, lastTsNs int64) error {
-		lastTime := time.Unix(0, lastTsNs)
-		glog.V(0).Infof("remote sync %s progressed to %v %0.2f/sec", *option.filerAddress, lastTime, float64(counter)/float64(3))
-		return remote_storage.SetSyncOffset(option.grpcDialOption, pb.ServerAddress(*option.filerAddress), option.bucketsDir, lastTsNs)
+	processor := NewMetadataProcessor(eachEntryFunc, 128)
```

Comment:
128 - Let's put it in a static variable and call it so that it is clear what it is responsible for. 
Also, maybe it’s worth connecting to the number core via GOMAXPROCS?

Reply:
I agree, but i actually tried to use the same logic from `filer.remote.sync`
But how is it possible to use GOMAXPROCS‌ for this code?

Reply:
Also I think merging this PR before the release will be helpful and I'll change the logic behind the concurrency value in future. Because it needs a change in both `filer.remote.gateway` and `filer.remote.sync`. 
In other hands I think founding the actual proper value for max concurrency parameter needs a separate discussion for setting it to a appropriate value.
So I will be very thankful if this PR gets merged.
Thanks.